### ISSUE TITLE
[MessageBar][ContrastRatio] Dismiss button for error message bar fails contrast ratio check

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#5013]
#### Describe the issue:
Error message bar dismiss button is not following contrast ratio guidelines

#### Actual behavior:
![image](https://user-images.githubusercontent.com/11937034/40612959-4154cf7c-629a-11e8-8ea6-b42ef040ecf9.png)

![image](https://user-images.githubusercontent.com/11937034/40612770-76b5ebd4-6299-11e8-994c-a90afc1aae17.png)

#### Expected behavior:

Contrast ratio should be at least 4.5:1

**Comment by betrue-final-final** [comment]
Good catch. The background changed to work with dark themes as well. We should change this to neutralPrimary.

**Comment by betrue-final-final** [comment]
@phkuo, knock on effect of changing the color. Can you fix this?
